scripts/docstrings.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, project_url=None):
 
     def process_docstring(self, docstring):
         docstring = docstring.replace("Args:", "# Arguments")
-        docstring = docstring.replace("Arguments:", "# Arguments")
+        docstring = docstring.replace("Args:", "# Arguments")
```

Comment:
Again, only accept the change on this line if https://github.com/keras-team/keras-contrib/pull/550 is accepted (oh, and change to `docstring.replace("Args:", "# Args")`)